packages/calling/src/CallingClient/calling/call.ts

```diff
@@ -908,6 +912,7 @@ export class Call extends Eventing<CallEventTypes> implements ICall {
    * @param event - Call Events.
    */
   private async handleOutgoingCallSetup(event: CallEvent) {
+    this.callState = CALL_EVENT_KEYS.SETUP;
```

Comment:
As discussed, please use `this.callStateMachine.state` and `this.callStateMachine.state.nextStates` to get current and next possible states instead of hardcoding possible states

Reply:
Updated